index.js

```diff
@@ -1,48 +1,53 @@
 'use strict'
 const json = require('fast-json-stable-stringify')
-const CID = require('cids')
 const isCircular = require('is-circular')
 const transform = require('lodash.transform')
-const codecInterface = require('@ipld/codec-interface')
 
-const _encode = obj => transform(obj, (result, value, key) => {
-  if (CID.isCID(value)) {
-    result[key] = { '/': value.toBaseEncodedString() }
-  } else if (Buffer.isBuffer(value)) {
-    result[key] = { '/': { base64: value.toString('base64') } }
-  } else if (typeof value === 'object' && value !== null) {
-    result[key] = _encode(value)
-  } else {
-    result[key] = value
-  }
-})
+module.exports = multiformats => {
+  const { CID, bytes, multibase } = multiformats
+  if (!multibase.has('base64')) throw new Error('Multibase must include base64')
+  if (!multibase.has('base32')) throw new Error('Multibase must include base32')
+  const _encode = obj => transform(obj, (result, value, key) => {
+    if (CID.isCID(value)) {
+      result[key] = { '/': value.toString() }
+    } else if (bytes.isBinary(value)) {
+      value = bytes.coerce(value)
+      result[key] = { '/': { base64: multibase.encode(value, 'base64') } }
```

Comment:
We changed the spec, the property is now called `bytes` and not `base64` anymore.

Reply:
will do